packages/es-client/tests/test-granule-upsert.js

```diff
@@ -305,13 +480,50 @@ test('upsertGranule does not update "running" granule to "failed" for same execu
     esClient,
     updates,
     index: esIndex,
+  }, false);
+  const updatedEsRecord = await t.context.esGranulesClient.get(granule.granuleId);
+  t.like(updatedEsRecord, updates);
```

Comment:
We should be comparing against the original `granule`, right? 🤔

Reply:
No - this test is validating the update *does* happen.

Reply:
Ah I understand now 😮‍💨 Not sure why I didn't the first time around